### ISSUE TITLE
TestStructuredMeshGeneratorProcessQuadrilateral fails in Release but works in FullDebug

### ISSUE THREAD

**Issue by philbucher** [KratosMultiphysics/Kratos#1440]
I get the following error on my machine:
~~~sh
    TestStructuredMeshGeneratorProcessQuadrilateral Failed with message: 
        Error: Check failed becuase total_area is not equal to 100.
in kratos/tests/processes/test_structured_mesh_generator_process.cpp:110:virtual void Testing::TestStructuredMeshGeneratorProcessQuadrilateral::TestFunction()
~~~

It works in FullDebug but fails in Release

**Comment by philbucher** [comment]
@pooyan-dadvand I saw that you made this test so I assigned you

**Comment by pooyan-dadvand** [comment]
Let me check

**Comment by pooyan-dadvand** [comment]
It works for me, would you please check if is a tolerance problem or uninitialized variable (by valgrind)

**Comment by philbucher** [comment]
fixed in #1446
